### ISSUE TITLE
[Bug]: Core keeper (ckserver) missing libxi6 as a dependency

### ISSUE THREAD

**Issue by TrythisTV** [GameServerManagers/LinuxGSM#4175]
### User story

Determine if libxi6 is needed on all instances of ckserver or if its just a weird fluke with my server

### Game

Core Keeper (ckserver)

### Linux distro

Ubuntu 20.04

### Command

command: start, command: debug

### Further information

When I run ./ckserver start after a successful install on Ubuntu 20.04.5 LTS the server would segfault, this was observed in logs, and also when running ./ckserver debug it would begin starting the server and then segfault.

I found on another github repo where they are making a docker container for core keeper server that a couple days ago there was a commit to "fix segmentation fault" and the commit added libxi6 as a dependency, I installed it via apt-get on my server, and running ./ckserver start or ./ckserver debug the server starts successfully and runs great.

I apologize that I am likely butchering every rule on formatting this report and giving you the information needed, its my first time sending a bug report on github.

### Relevant log output

```shell
it appears my log was overwritten after installing libxi6 and starting the server
```

### Steps to reproduce

1: Install ckserver with official lgsm method on headless ubuntu server without any gui type X server stuff installed, including libxi6
2: Attempt to start server with ./ckserver start or ./ckserver debug
3: Observe segmentation fault
4: Install libxi6
5: Start server with ./ckserver start and observe that server starts without any issue

**Comment by seedstohr** [comment]
thank you. i was having the same seg fault and this helped get the server running.

**Comment by TheBlueTroll** [comment]
I can additionally confirm that following normal install instructions and then `apt install libxi6` also fixed for me on Ubuntu 23.04.
otherwise i was getting a segfault in the server log.

**Comment by dgibbs64** [comment]
ae4cc3b699730ea3250b3493298460eaf63ed9c6
